Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -148,8 +148,8 @@
 - type: entity
   parent: ClothingOuterHardsuitBaseNFNfsd
   id: ClothingOuterHardsuitNfsdBronze
-  name: NFSD bronze patrol hardsuit
-  description: A special suit that protects from the danger of space, employed by NFSD patrol officers. Not certified to be blunderbuss-proof.
+  name: NFSD bronze patrol suit
+  description: A lightly armored suit that protects from the danger of space, standard issue to be worn by NFSD on patrol. This bronze variant has cadet rank markings.
```

```suggestion
  description: The standard-issue NFSD space proof suit for patrol duty. This bronze variant has cadet rank markings.
```
Shorter.  Not a ton of value in saying that standard-issue things are light or heavy if they're the basis for comparison.